src/redux/actions/clusters.js

```diff
@@ -153,21 +152,23 @@ export function fetchCluster(id) {
 }
 
 export function fetchClusters(type) {
-  const action = netActions.addNetworkCall('fetch_clusters', 'Retreive clusters');
-  dispatch(pendingNetworkCall(true));
-  client.listClusters(type)
-    .then(
-      resp => {
-        dispatch(netActions.successNetworkCall(action.id, resp));
-        dispatch(updateClusters(resp.data));
-        dispatch(pendingNetworkCall(false));
-      },
-      err => {
-        dispatch(netActions.errorNetworkCall(action.id, err));
-        dispatch(pendingNetworkCall(false));
-      });
+  return dispatch => {
+    const action = netActions.addNetworkCall('fetch_clusters', 'Retreive clusters');
+    dispatch(pendingNetworkCall(true));
+    client.listClusters(type)
+      .then(
+        resp => {
+          dispatch(netActions.successNetworkCall(action.id, resp));
+          dispatch(updateClusters(resp.data));
+          dispatch(pendingNetworkCall(false));
+        },
+        err => {
+          dispatch(netActions.errorNetworkCall(action.id, err));
+          dispatch(pendingNetworkCall(false));
+        });
 
-  return action;
+    return action;
+  };
 }
```

Comment:
I assume these are just formatting fixes? In future it might be go to do the formatting changes in a separate commit, to make it easier to see the logic changes.